tests/ha/check_after_reboot.pm

```diff
@@ -27,6 +27,9 @@ sub run {
     $timeout_scale = 2 if ($timeout_scale < 2);
     set_var('TIMEOUT_SCALE', $timeout_scale) unless (is_x86_64);
 
+    # iptables is not installed in SLE 16 by default
+    zypper_call 'in iptables' if is_sle('>=16');
```

Comment:
I don't think this is necessary. The `iptables` command in L44 only runs when **QDEVICE_TEST_ROLE** is set to `client`, and in those cases the `iptables` package would be installed by the `ha/qnetd` module as modified in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21992